CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ The sections should follow the order `Added`, `Changed`, `Fixed`, and `Removed`.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+
+- On macOS, use the `AppleFontSmoothing` user default to decide whether fonts should be "smoothed"
+
+### Changed
+
+- Renamed `darwin::Rasterizer` to `darwin::CoreTextRasterizer`
+
+### Fixed
+
+- On macOS, `use_thin_strokes` and `set_font_smoothing` do nothing in Big Sur and Monterey
```

Comment:
Due to the present tense "do nothing" here, it's not entirely clear here if this is how it works now, or if this is what got fixed.

Reply:
Will clarify, thanks.